### PR TITLE
Pulumi Release Notes Blog

### PR DESCRIPTION
**Pull request by arunkumar611**
To-do
- Get updates from Core team
- Get updates from providers team

### CODE REVIEW

File: content/blog/pulumi-release-notes-106/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+{{< video title="Using pulumi new with the new AI option" src="/blog/continue-on-error/coe.mp4" controls="false" autoplay="true" loop="true" >}}
+
+### Run Pulumi with Any TypeScript Version
+Previously, Pulumi's seamless TypeScript experience was limited to version 3.8, requiring manual build steps for newer versions. But now, with the release of the [Pulumi Node.js SDK 3.113.0](https://www.npmjs.com/package/@pulumi/pulumi), you can use any TypeScript version 3.8 or later, including the latest [TypeScript 5.4](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-4.html), by simply adding it as a dependency in your project's package.json file. Pulumi will automatically use the specified version for compilation, unlocking access to the newest TypeScript features and improved type checking. For backward compatibility and ease of use within monorepos, TypeScript and ts-node are optional peer dependencies and Pulumi defaults to bundled versions of TypeScript 3.8 and ts-node 7 if no specific versions are declared. Read the [blog post](/blog/typescript-versions/) for a detailed example and instructions on upgrading your projects.
```
Comment (justinvp) [inline content/blog/pulumi-release-notes-106/index.md:115]: I'd just cut this sentence:

```suggestion
Previously, Pulumi's seamless TypeScript experience was limited to version 3.8, requiring manual build steps for newer versions. But now, with the release of the [Pulumi Node.js SDK 3.113.0](https://www.npmjs.com/package/@pulumi/pulumi), you can use any TypeScript version 3.8 or later, including the latest [TypeScript 5.4](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-4.html), by simply adding it as a dependency in your project's package.json file. Pulumi will automatically use the specified version for compilation, unlocking access to the newest TypeScript features and improved type checking. Read the [blog post](/blog/typescript-versions/) for a detailed example and instructions on upgrading your projects.
```

Alternatively, say something like. "New Pulumi TypeScript projects depend on the latest version of TypeScript by default.".

**Review by justinvp — APPROVED** [review]
One suggestion, but LGTM otherwise

File: content/blog/pulumi-release-notes-106/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+## Providers and Packages
+
+### Next-Generation Docker Image Builds with Pulumi
+Pulumi introduces the new [Docker Build provider](/registry/packages/docker-build/), a dedicated package designed to modernize and streamline Docker image builds directly from your Pulumi programs. This new provider, built upon the widely adopted Docker provider, empowers developers to leverage the latest Docker capabilities and BuildKit features, enabling more efficient and robust image creation workflows. The new provider exposes Docker's next-generation buildx interface, unlocking advanced features such as multi-platform image support, advanced caching mechanisms, built-in support for build secrets, and seamless integration with Docker Build Cloud. This unlocks powerful scenarios, including building multi-architecture images and optimizing build performance with various cache backends. Read the [blog post](/blog/docker-build/) for detailed examples and migration guides.
```
Comment (mjeffryes) [inline content/blog/pulumi-release-notes-106/index.md:120]: I would drop "This new provider, built upon the widely adopted Docker provider, empowers developers to leverage the latest Docker capabilities and BuildKit features, enabling more efficient and robust image creation workflows." This provider is independent of the previous Docker provider.

**Review by mjeffryes — APPROVED** [review]
Thank you!

**Review by meagancojocar — APPROVED** [review]

File: content/blog/pulumi-release-notes-106/index.md (hunk 2 above)
Comment (arunkumar611) [reply content/blog/pulumi-release-notes-106/index.md]: updated. Thank you!

**Pull request merged**